src/office/mediaFiles.ts

```diff
@@ -48,8 +48,7 @@ export class MediaFiles {
         // generate unique media file name
         const extension = MimeTypeHelper.getDefaultExtension(mime);
         do {
-            this.nextFileId++;
-            path = `${MediaFiles.mediaDir}/media${this.nextFileId}.${extension}`;
+            path = `${MediaFiles.mediaDir}/image${++this.nextFileId}.${extension}`;
         } while (this.hashes[path]);
```

Comment:
This line is supposed to prevent the issue you are experiencing. We should check and see why it doesn't. See my comment below for more details.